graph/resolver/schema.resolvers.go

```diff
@@ -13,10 +13,13 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func (r *mutationResolver) AddDiscussionParticipant(ctx context.Context, discussionID string, userID string, discussionParticipantInput model.AddDiscussionParticipantInput) (*model.Participant, error) {
-	participantObj, err := r.DAOManager.CreateParticipantForDiscussion(ctx, discussionID, userID, discussionParticipantInput)
+func (r *mutationResolver) AddDiscussionParticipant(ctx context.Context, discussionID string, discussionParticipantInput model.AddDiscussionParticipantInput) (*model.Participant, error) {
+	authedUser := auth.GetAuthedUser(ctx)
```

Comment:
align on `authedUser`

Reply:
Do you think we should check whether the user is the moderator or sth?